client/scss/components/_header.scss

```diff
@@ -0,0 +1,289 @@
+//* @define header; weak
+
+.header {
+  padding-top: 0.8rem;
+}
+
+.header__upper-wrap {
+  border-bottom: 1px solid color('keyline-grey');
+}
+
+.header__upper,
+.header__lower {
+  display: flex;
+}
+
+.header__upper {
+  padding-bottom: 0.8rem;
+  justify-content: flex-end;
+  position: relative;
+
+  @include respond-to('medium') {
+    justify-content: space-between;
+  }
+}
+
+.header__lower {
+  justify-content: space-between;
+  position: relative;
+
+  &.is-active {
```

Comment:
There would still need to be some level of nesting in order to establish that the generic state class (e.g. `is-active`) is applied to the element that we're interested in – that's the trade off. Otherwise, you could have a different element that is a parent of the element we're interested in that also had the `is-active` class that would trigger a change in styles that we didn't want.

```html
<body class="is-active">
  <h1>Contrived example</h1>
  <div class="thingy is-active">
    <h2>Thingy heading</h2>
    <div class="thingy__drawer">
      <p>Only display this `thingy__drawer` el when `thingy` has an `is-active` class, not when e.g. the `body` has an `is-active` class.
    </div>
  </div>
</body>
```